### PR TITLE
post about chapters and tech leads

### PR DESCRIPTION
**Pull request by noheadx**
Change post date to any date in the past for local testing

### CODE REVIEW

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+![Differences between Chapter Leads and Product Owners]({{site.baseurl}}/assets/img/220930-chapter/how_what.png)
+*Difference between Chapter Lead and Product Owner ([Henrik Kniberg & Anders Ivarsson, Oct 2012](https://blog.crisp.se/wp-content/uploads/2012/11/SpotifyScaling.pdf))*
+
+## What's a Chapter & Chapter Lead for us?
+"The Chapter's more of a regular's table for its members, a place where they can touch base with like-minded and can discuss technical things. The Chapter also defines its guidelines regarding work processes (documentation, coding, etc.)."
+
+We define a chapter as a group of people that
+- shares similar expertise (or is interested in it) and
+- shares knowledge, experience, and best practices internally and with other chapters and
+- is led by a tech lead and
```
Comment (Arkinator) [inline _posts/2022-09-30-chapter-tech-leads.markdown:36]: "tech lead" might be somewhat a loose term (and maybe not technically correct). "a very senior developer"?

Comment (Arkinator) [reply _posts/2022-09-30-chapter-tech-leads.markdown:36]: thats bull as well... :/

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+- has a size of max. 10 regular employees
+
+There are many definitions for what a Chapter Lead (CL) is and what is required to become one. For us, we defined it as a senior developer who can 
+- create a vision for technical topics,
+- mentor other developers, 
+- pass the passion for development to the members, and
```
Comment (Arkinator) [inline _posts/2022-09-30-chapter-tech-leads.markdown:42]: pass => ignite

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+"The Chapter's more of a regular's table for its members, a place where they can touch base with like-minded and can discuss technical things. The Chapter also defines its guidelines regarding work processes (documentation, coding, etc.)."
+
+We define a chapter as a group of people that
+- shares similar expertise (or is interested in it) and
+- shares knowledge, experience, and best practices internally and with other chapters and
+- is led by a tech lead and
+- has a size of max. 10 regular employees
+
+There are many definitions for what a Chapter Lead (CL) is and what is required to become one. For us, we defined it as a senior developer who can 
+- create a vision for technical topics,
```
Comment (Arkinator) [inline _posts/2022-09-30-chapter-tech-leads.markdown:40]: again: might be preference, but the commas and "and" break my flow

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+- mentor other developers, 
+- pass the passion for development to the members, and 
+- organizes joint efforts for education and training.
```
Comment (Arkinator) [inline _posts/2022-09-30-chapter-tech-leads.markdown:43]: "who can organizes" => organize

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+- is led by a tech lead and
+- has a size of max. 10 regular employees
+
+There are many definitions for what a Chapter Lead (CL) is and what is required to become one. For us, we defined it as a senior developer who can 
+- create a vision for technical topics,
+- mentor other developers, 
+- pass the passion for development to the members, and 
+- organizes joint efforts for education and training. 
+
+For that, the CL is free to organize the Chapter and communication as seems necessary.<br/>
```
Comment (Arkinator) [inline _posts/2022-09-30-chapter-tech-leads.markdown:45]: CL => Chapter Lead (even if defined above, not a common abbreviation)

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+ He (there are only male leads in our case at the moment) would be responsible for
```
Comment (Arkinator) [inline _posts/2022-09-30-chapter-tech-leads.markdown:46]: would be => is

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+- create a vision for technical topics,
+- mentor other developers, 
+- pass the passion for development to the members, and 
+- organizes joint efforts for education and training. 
+
+For that, the CL is free to organize the Chapter and communication as seems necessary.<br/>
+ He (there are only male leads in our case at the moment) would be responsible for 
+ - creating the vision for the Chapter
+ - setting up and maintaining  the Chapter Backlog
+ - for supporting team members in becoming experts in the Chapter's mission topic.
```
Comment (Arkinator) [inline _posts/2022-09-30-chapter-tech-leads.markdown:49]: "for supporting" => "supporting"

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+A Chapter Lead would not decide on salary rises, approve vacation days or hire people. But a CL will always give valuable input and information regarding these topics to the line manager.
```
Comment (Arkinator) [inline _posts/2022-09-30-chapter-tech-leads.markdown:51]: would not => does not

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+## How did we do the setup?
+
+### 1. Cut the elephant into pieces
+Once the goal was decided, we had to figure out how to get there, and the first step was to cut out the chapters.
```
Comment (Arkinator) [inline _posts/2022-09-30-chapter-tech-leads.markdown:56]: cut out => either "carve out" or "define"

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+I spoke to each candidate personally, and everyone was immediately on fire for the position, shooting ideas of what could be done and achieved in this setting.
+That showed we were on the right track! 
+Every candidate received a "First 100 days expectation"-email giving scope for the role. It is important to emphasize that the position was created as a tech lead. No disciplinary management tasks were involved. The message was: "That's your topic and your people - you are responsible for growing both with any support from management that it takes!"
+
+### 3. Start the work
+To create an environment for the chapter leads and developers where the education and growth of knowledge could bloom, we had to make a few yet fundamental decisions about the freedom of the Chapter. Some issues were easy as they were clearly defined by our company guidelines, e.g., who approves vacation requests. Others required deeper discussion like educational budget or flat rate time for chapter work.
```
Comment (Arkinator) [inline _posts/2022-09-30-chapter-tech-leads.markdown:69]: "few yet fundamental decisions about" => "few, yet fundamental, decisions about" (unsure, im not good at this...)

Comment (Arkinator) [reply _posts/2022-09-30-chapter-tech-leads.markdown:69]: "flat rate time" = "time allotment"

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+---
+
+At gematik, we work in a matrix organization like many medium or large companies. The idea of a matrix organization is to group different skills in vertical cross-functional product teams where the work is done (the What) while departments, like Software Development, are taking care of the technical education of their employees (the How). 
+
+![Sketch of the matrix organization @gematik]({{site.baseurl}}/assets/img/220930-chapter/matrix_gematik.png)
+*Sketch of the matrix organization @gematik (gematik GmbH)*
+
+While this model provides an undeniable scaling effect, it bears some challenges that should be taken into account:
+- While the teams may work on different products, they often face the same issues during development. We needed to make sure to have a cross-product exchange of solutions
+- Our teams mostly use the same essential technologies, Java, SpingBoot, Maven, and REST APIs - only very few teams are really unique in their tech stack, like the mobile app teams. We need to ensure that people are skilled and trained in the required techs and that we can help across teams.
```
[the remark this answers is not in the document]
Comment (noheadx) [reply _posts/2022-09-30-chapter-tech-leads.markdown:18]: The listed technologies are essential to our work as is - basic is too weak from my pov

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+- Teams have different sizes depending on their needs. We have teams with only 1 developer, so we had to provide a home base for these people.
```
[the remark this answers is not in the document]
Comment (noheadx) [reply _posts/2022-09-30-chapter-tech-leads.markdown:19]: As "needed" above it depends on the narrator's perspective - the article is about a change that's already done and one could argue that people now have a home base - so past tense is actually correct.
Anyways, for the sake of consistency, I changed it to present tense.

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+- The future need for skills in a team depends on its product evolvement, but in the matrix, there is a natural disconnect between the product team and department. So, we more often reacted to a new demand instead of being prepared.
+This is why the Software Development department recently switched its structure.
+Previously, we had 2 teams covering 20 people each that supposedly focussed on different topics trying to avoid the caveats described above.
+This resulted in 2 team leads giving their best in keeping the balls in the air but eventually losing focus on what the company needed.
```
[the remark this answers is not in the document]
Comment (noheadx) [reply _posts/2022-09-30-chapter-tech-leads.markdown:23]: yes, I had the same feeling while writing it but wanted to test reactions on it.
I removed the "leads" and left only "teams" as I see it as a shared responsibility

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+Therefore, we decided to give up the team structure and form a chapter structure (as explained in the (in-)famous Spotify model) around Technical Leads as Chapter Leads to structure the How for the product teams more efficiently.
+
+![Differences between Chapter Leads and Product Owners]({{site.baseurl}}/assets/img/220930-chapter/how_what.png)
+*Difference between Chapter Lead and Product Owner ([Henrik Kniberg & Anders Ivarsson, Oct 2012](https://blog.crisp.se/wp-content/uploads/2012/11/SpotifyScaling.pdf))*
+
+## What's a Chapter & Chapter Lead for us?
+"The Chapter's more of a regular's table for its members, a place where they can touch base with like-minded and can discuss technical things. The Chapter also defines its guidelines regarding work processes (documentation, coding, etc.)."
+
+We define a chapter as a group of people that
+- shares similar expertise (or is interested in it) and
```
[the remark this answers is not in the document]
Comment (noheadx) [reply _posts/2022-09-30-chapter-tech-leads.markdown:34]: true - it was unformatted text before and I switched to bullet points for improved readability

File: _posts/2022-09-30-chapter-tech-leads.markdown (hunk 1 above)
Comment (noheadx) [reply _posts/2022-09-30-chapter-tech-leads.markdown:36]: which is why I added a paragraph for explaining our definition

File: _posts/2022-09-30-chapter-tech-leads.markdown (hunk 3 above)
Comment (noheadx) [reply _posts/2022-09-30-chapter-tech-leads.markdown:40]: I think it is preference but I removed them for consistency

File: _posts/2022-09-30-chapter-tech-leads.markdown (hunk 2 above)
Comment (noheadx) [reply _posts/2022-09-30-chapter-tech-leads.markdown:42]: spark? Actually, I don't want to use any term indicating that the members are not passionate for development - it's more about nurturing the flame and making it bigger - if we need to ignite/spark it we have a different issue

File: _posts/2022-09-30-chapter-tech-leads.markdown (hunk 9 above)
Comment (noheadx) [reply _posts/2022-09-30-chapter-tech-leads.markdown:56]: yes, but I wanted to put a reference to the "cut the elephant" - let's keep it

File: _posts/2022-09-30-chapter-tech-leads.markdown (hunk 10 above)
Comment (noheadx) [reply _posts/2022-09-30-chapter-tech-leads.markdown:69]: The term "time allotment" misses the "blanket" character, what we discussed was a certain amount of time that is just reserved without any justification.

File: _posts/2022-09-30-chapter-tech-leads.markdown [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+
+![Sketch of the matrix organization @gematik]({{site.baseurl}}/assets/img/220930-chapter/matrix_gematik.png)
+*Sketch of the matrix organization @gematik (gematik GmbH)*
+
+While this model provides an undeniable scaling effect, it bears some challenges that should be taken into account:
+- While the teams may work on different products, they often face the same issues during development. We needed to make sure to have a cross-product exchange of solutions
+- Our teams mostly use the same essential technologies, Java, SpingBoot, Maven, and REST APIs - only very few teams are really unique in their tech stack, like the mobile app teams. We need to ensure that people are skilled and trained in the required techs and that we can help across teams.
+- Teams have different sizes depending on their needs. We have teams with only 1 developer, so we had to provide a home base for these people.
+- The future need for skills in a team depends on its product evolvement, but in the matrix, there is a natural disconnect between the product team and department. So, we more often reacted to a new demand instead of being prepared.
```
[the remark this answers is not in the document]
Comment (noheadx) [reply _posts/2022-09-30-chapter-tech-leads.markdown]: ah grammarly - thou artst the finest of all tools! :)

**Pull request merged**